### ISSUE TITLE
Error when trying a simple copy

### ISSUE THREAD

**Issue by theirishpenguin** [saygoweb/ansible-plugin-copyv#4]
Have you encountered the error the following error?

```
TypeError: _make_tmp_path() takes exactly 2 arguments (1 given)
```

It happened to when when testing a simple copyv with src and dest arguments supplied.

**Comment by cambell-prince** [comment]
Can you post a stack trace that includes the error.  Also, confirm that you are using the plugin from the ansible-2.x branch rather than the master branch (which will fail badly when used with Ansible 1.x).

**Comment by theirishpenguin** [comment]
Yes I'm using the ansible-2.x branch and the full stacktrace is...

```
An exception occurred during task execution. The full traceback is:
Traceback (most recent call last):
  File "/home/foo/ansible/lib/ansible/executor/task_executor.py", line 124, in run
    res = self._execute()
  File "/home/foo/ansible/lib/ansible/executor/task_executor.py", line 426, in _execute
    result = self._handler.run(task_vars=variables)
  File "/home/foo/ansible_project/playbooks/my/action_plugins/copyv.py", line 221, in run
    tmp = self._make_tmp_path()
TypeError: _make_tmp_path() takes exactly 2 arguments (1 given)

fatal: [server.example.com]: FAILED! => {"failed": true, "msg": "Unexpected failure during module execution.", "stdout": ""}
```

**Comment by anarcher** [comment]
In ansible `2.1.0.0` , I got  same error. The version of Ansible may change `ActionBase._make_tmp_path` ..  I change it like below.  I think It works.. 

```
>         remote_user = task_vars.get('ansible_ssh_user') or self._play_context.remote_user
147c148
<                 tmp = self._make_tmp_path()
---
>                 tmp = self._make_tmp_path(remote_user)
221c222
<                         tmp = self._make_tmp_path()
---
>                         tmp = self._make_tmp_path(remote_user)
```

**Comment by cambell-prince** [comment]
This plugin won't work with Ansible 2.1.x and shouldn't be required.  This feature should now be implemented in Ansible 2.1.x via [PR 15417](https://github.com/ansible/ansible/pull/15417)
